src/controller/index.ts

```diff
@@ -306,7 +272,7 @@ export default class SimulariumController {
                     connectionParams.clientSimulatorParams,
                     connectionParams.simulariumFile
                 );
-                this.networkEnabled = true;
+                this.networkEnabled = true; // This confuses me, because local files also go through this code path
```

Comment:
good point, this could be targeted for removal soon. It might be serving as "are we using a RemoteSimulator or not" but I 'm not sure.